Sources/MongoSwift/WriteConcern.swift

```diff
@@ -3,6 +3,28 @@ import Foundation
 
 /// A class to represent a MongoDB write concern.
 public struct WriteConcern: Codable {
+    /// Majority WriteConcern
```

Comment:
I don't mean to hammer the point, but I really think this docstring needs to be updated to mention that `wtimeout` and `j` are unset.

Reply:
No worries about 🔨 I agree it can be clearer